snakePipes/workflows/ATAC-seq/ATAC-seq

```diff
@@ -39,12 +39,12 @@ def parse_args(defaults={"verbose": False, "configfile": None, "cluster_configfi
                           type=int,
                           default=defaults["bw_binsize"])
 
-    optional.add_argument("--atac-fragment-cutoff",
-                          dest="atac_fragment_cutoff",
+    optional.add_argument("--atac-fragment-size-cutoff",
+                          dest="atac_fragment_size_cutoff",
```

Comment:
could you change this to --fragSize_cutoff or at least `fragmentSize_cutoff` to match the other variable `fragmentCount_cutoff`?